api/app/controllers/api/v1/output_csv_controller.rb

```diff
@@ -503,23 +503,60 @@ def output_public_relations_csv
     end
     bom = "\uFEFF"
     csv_data = CSV.generate(bom) do |csv|
-      column_name = %w(参加団体名 PR文 URL アナウンス有無)
+      column_name = %w(参加団体名 PR文 PR画像)
       csv << column_name
-      @public_relations.each do |public_relations|
-        # データが存在しない場合はスキップする
-        if public_relations.nil?
+      @public_relations.each do |public_relation|
+        if public_relation.nil?
           next
         end
         column_values = [
-          public_relations.group.name,
-          public_relations.blurb,
-          public_relations.picture_path,
-          public_relations.is_announcement_requested ? "有" : "無",
+          public_relation.group.name,
+          public_relation.pr_comment,
+          public_relation.pr_image,
         ]
         csv << column_values
       end
     end
     send_data(csv_data, filename:"参加団体PR申請_#{filename_year}年度.csv")
   end
 
+  def output_fire_equipment_orders_csv
+    if params[:fes_year_id].to_i == 0
+      @fire_equipment_orders = FireEquipmentOrder.all
+      filename_year = "全"
+    else
+      fes_year = FesYear.find(params[:fes_year_id])
+      @fire_equipment_orders = FireEquipmentOrder.joins(:group).where(groups: { fes_year_id: fes_year.id })
+      filename_year = fes_year.year_num
+    end
+    bom = "\uFEFF"
+    csv_data = CSV.generate(bom) do |csv|
+      column_name = %w(ID 団体名 品目 数量 燃料 使用場所 持ち帰り 備考)
```

Comment:
figmaだと`火気の名称` , `火気の台数` , `使用用途` だった

```suggestion
      column_name = %w(ID 団体名 火気の名称 火気の台数 燃料 使用用途 持ち帰り 備考)
```